cmd/whisperImporter/whisperReader/main.go

```diff
@@ -0,0 +1,298 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/kisielk/whisper-go/whisper"
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/cmd/whisperImporter"
+	"github.com/raintank/metrictank/mdata/chunk"
+	"gopkg.in/raintank/schema.v1"
+)
+
+var (
+	whisperDirectory = flag.String(
+		"whisper-directory",
+		"/opt/graphite/storage/whisper",
+		"The directory that contains the whisper file structure",
+	)
+	chunkSpanStr = flag.String(
+		"chunkspan",
+		"10min",
+		"duration of chunks",
+	)
+	namePrefix = flag.String(
+		"name-prefix",
+		"",
+		"Prefix to prepend before every metric name",
```

Comment:
we should mention that if they want a dot between prefix and the metric name, they should add it to the prefix themselves.  there's always this same confusing nuance about any graphite tool that has a prefix option.